airbyte-integrations/bases/standard-test/standard_test/tests/test_core.py

```diff
@@ -51,6 +50,13 @@ def test_check(self, connector_config, docker_runner: ConnectorRunner):
         assert len(con_messages) == 1, "Connection status message should be emitted exactly once"
         assert con_messages[0].connectionStatus.status == Status.SUCCEEDED
 
+    def test_check_without_network(self, connector_config, docker_runner: ConnectorRunner):
```

Comment:
my guess is that this check will work well for most sources but sources which don't always require internet like source-file might not need it. I see a couple of ways around this: 

1. add a flag to the config `requires_internet` that can be used to disable this check
2. don't use this check and rely on the other ones which malform the configs

Reply:
I vote for №2